src/app/components/activity/feed/new-button/new-button.vue

```diff
@@ -9,10 +9,10 @@
 <style lang="stylus" scoped>
 @require '../variables'
 @require '~styles-lib/mixins'
+@require '~styles/variables'
```

Comment:
this isn't needed, the first variables line imports it in

Reply:
👌